content/blogposts/2022/release-4.2.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Sourcegraph 4.2 release"
+publishDate: 2022-11-29T10:00-07:00
+description: "Sourcegraph 4.2 introduces the new audit log, executor secrets for server-side Batch Changes, and Smart Search to make it easier to find the search results you're looking for."
+tags: [blog, release]
+slug: "release/4.2"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.2/sourcegraph-4-2-hero.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.2/sourcegraph-4-2-hero.png
+changelogItems:
+  - description: The search input has a new recent searches button, and recent searches can be cycled via the up/down arrow keys for quick access to previous searches.
+    url: https://github.com/sourcegraph/sourcegraph/pull/44544
+    category: Search
+  - description: "When the `content:` filter is used in a query, it now behaves more predictably by only searching file contents. Previously, file contents, paths, and repos were searched. However, as before, if `type:` is also set, the `content:` filter will search for results of the specified `type:`."
```

Comment:
@iskyOS - is repos referring to repositories or the "repo" filter? If the former, it should be repositories.